### PR TITLE
Remove images options in oadm command examples

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

File: README_OSE.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ ansible_ssh_user=root
 deployment_type=enterprise
 
 # Pre-release registry URL
-oreg_url=rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}
+oreg_url=registry.access.redhat.com:5001/openshift3/ose-${component}:${version}
```
Comment (sdodson) [inline README_OSE.md]: Remove the 5001 port?

File: README_OSE.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ On the master host:
 ```sh
 oadm router --create=true \
   --credentials=/etc/openshift/master/openshift-router.kubeconfig \
-  --images='rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}'
+  --images='registry.access.redhat.com:5001/openshift3/ose-${component}:${version}'
```
Comment (sdodson) [inline README_OSE.md]: Remove the 5001 port?

File: README_OSE.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ On the master host:
 ```sh
 oadm registry --create=true \
   --credentials=/etc/openshift/master/openshift-registry.kubeconfig \
-  --images='rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}' \
+  --images='registry.access.redhat.com:5001/openshift3/ose-${component}:${version}' \
```
Comment (sdodson) [inline README_OSE.md]: Remove the 5001 port? Also as of 3.0.1 this shouldn't be necessary, perhaps remove the entire --images line?

File: inventory/byo/hosts.example [hunk 4]
```diff
@@ -24,7 +24,7 @@ deployment_type=atomic-enterprise
 #use_cluster_metrics=true
 
 # Pre-release registry URL
-#oreg_url=rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}
+#oreg_url=registry.access.redhat.com:5001/openshift3/ose-${component}:${version}
```
Comment (sdodson) [inline inventory/byo/hosts.example]: I'd leave it as is as it's technically correct however that registry isn't accessible externally. Perhaps we should change this to be
```
# Alternate image format string, useful for internal mirrors
#oreg_url=example.com/openshift3/ose-${component}:${version}
```

**Comment by nak3** [comment]
@sdodson Yes, you are right.Thank you. I fixed them.

**Comment by sdodson** [comment]
LGTM, just rebase

**Comment by nak3** [comment]
OK, rebased.

**Comment by abutcher** [comment]
@sdodson @nak3 This readme is weird since it's titled "Installing OSEv3 from dev puddles using ansible". Should we have a public OSE readme that is separate and tuck this one away? Or should we say, "Hey! the real docs are over here < link >" in the top level readme and leave the internal bits in this one?

**Comment by sdodson** [comment]
@abutcher I agree, we should get rid of discussion of internal processes and references to internal hostnames. We'd be better off hosting a set of inventories that set the right values on internal resources.

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by sdodson** [comment]
LGTM :+1:  @brenton docs changes only here, ready to merge

**Pull request merged**